src/main/java/games/strategy/engine/persistence/serializable/PersistenceDelegateRegistryFactory.java

```diff
@@ -0,0 +1,28 @@
+package games.strategy.engine.persistence.serializable;
+
+import games.strategy.internal.persistence.serializable.PropertyBagMementoPersistenceDelegate;
+import games.strategy.internal.persistence.serializable.VersionPersistenceDelegate;
+import games.strategy.persistence.serializable.DefaultPersistenceDelegateRegistry;
+import games.strategy.persistence.serializable.PersistenceDelegateRegistry;
+import games.strategy.util.Version;
+import games.strategy.util.memento.PropertyBagMemento;
+
+/**
+ * Factory for creating pre-configured instances of {@link PersistenceDelegateFactory}.
+ */
+public final class PersistenceDelegateRegistryFactory {
+  private PersistenceDelegateRegistryFactory() {}
+
+  /**
+   * Creates a new {@link PersistenceDelegateRegistry} that has been configured with all platform persistence delegates.
+   *
+   * @return A new {@link PersistenceDelegateRegistry}; never {@code null}.
+   */
+  public static PersistenceDelegateRegistry newPlatformPersistenceDelegateRegistry() {
```

Comment:
In general, I think this is a smell.  It could result in a dependency from the persistence layer to the orchestration layer (it's a bit hard to see at this point because there's so many different concerns under `g.s.engine.*`).

If the delegate were to self-register with the registry, wouldn't there be something akin to a chicken-and-egg problem?  If the registration call is in the delegate class, what triggers it?  I don't think it will work if you put it in the class initializer because the class initializer is only called when the class is loaded.  If no external code references the delegate class, I don't see how the class will ever be loaded.  Can you provide a code sample of what you're thinking if I'm misinterpreting your suggestion here?

As I mentioned before, if TripleA were running in a DI container, I would set this up using a Whiteboard pattern, which is probably what you are thinking about emulating.  The difference in that case is you have a third party (the Whiteboard service) registering the delegates with the registry, which not only solves the self-registration chicken-and-egg problem, but it breaks any dependency in the direction from the persistence layer to higher layers.